tests/test_elog.py

```diff
@@ -68,6 +76,15 @@ def test_elog_post(mockelog):
         mockelog.post('Failure', experiment=False, facility=False)
 
 
+def test_elog_set(mockelog):
+    msg = 'set method for post'
+    st = mockelog.set(msg)
+    time.sleep(0.1)  # needed to allow status to resolve
```

Comment:
Cleaner is probably to wait on the status
```suggestion
    st.wait(timeout=0.1)
```

Reply:
I actually don't understand why I have to wait at all, since I set the status to be done and successful to start with.  I'm fine with this change though.